js/personal-settings.js

```diff
@@ -0,0 +1,12 @@
+$(document).ready(() => {
```

Comment:
Please don't use the global `$` but add another webpack entry point for this https://blog.wuc.me/2019/06/12/problem-nextcloud-global-dependencies.html